modules/virt-measuring-latency-vm-secondary-network.adoc

```diff
@@ -4,13 +4,18 @@
 
 :_content-type: PROCEDURE
 [id="virt-measuring-latency-vm-secondary-network_{context}"]
-= Checking network connectivity and latency for virtual machines on a secondary network
+= Virtual machine latency checkup
 
-You use a predefined checkup to verify network connectivity and measure latency between two virtual machines (VMs) that are attached to a secondary network interface.
+You use a predefined checkup to verify network connectivity and measure latency between two virtual machines (VMs) that are attached to a secondary network interface. The latency checkup uses the ping utility.
 
-To run a checkup for the first time, follow the steps in the procedure.
+You run a latency checkup by performing the following steps:
 
-If you have previously run a checkup, skip to step 5 of the procedure because the steps to install the framework and enable permissions for the checkup are not required.
+. Create a service account, role, and role bindings to provide cluster access permissions to the latency checkup.
+. Create a config map. The config map provides the input to run the checkup and stores the results.
```

Comment:
s/Create a config map. The config map provides/Create a config map to provide (minimalism)